### DISCUSSION TITLE
DigitalOcean App Platform HTTPS

### DISCUSSION THREAD

**Discussion by kieranbarlow** [statamic/cms#7116] [Q&A]
I've deployed my application via DigitalOcean App Platform which forces https.

When viewing the site via my domain it works as expected however when accessing the backend /cp I get various mixed content notifications in my console log and the admin panel doesn't work. 

I'm not sure why the resources for the backend are being requested as http rather than https.

**Comment by edalzell** [comment]
What is your `APP_URL` set to and please share your `sites.php`

**Comment by kieranbarlow** [comment]
My APP_URL is set to my domain, which was the same as listed in the sites.php

My sites.php contains the default.

```
'sites' => [

      'default' => [
          'name' => config('app.name'),
          'locale' => 'en_GB',
          'url' => '/',
      ],

  ],
```

I've tried adding my domain to the url but no joy.

**Reply by kieranbarlow** [reply]
Solved, I had to set 

`protected $proxies = "*"; `

In my TrustProxies.php file because I'm using Cloudflare

**Reply by edalzell** [reply]
You could also solve it by using your `APP_URL` in `sites.php` and setting the domain to `https://....`. This is because CF is connecting on port 80 cuz your site will respond there.

**Comment by jasonvarga** [comment]
If you are using Cloudflare, you could also use https://github.com/monicahq/laravel-cloudflare rather than setting `$proxies = '*'`.

That package will update your trusted proxies to Cloudflare servers automatically.
